### DISCUSSION TITLE
Mujoco .skn import error insufficient data in skn file

### DISCUSSION THREAD

**Discussion by PiNo010** [google-deepmind/mujoco#1964] [Asking for Help]
Hi,

I'm a student and I'm trying to use MuJoCo for a university project.
We have to model an hand and import in mujoco to make it move.
I'm having problem in implementing the .skn file in the hand.xml. To make the .skn file i used the script in 
https://github.com/google-deepmind/dm_control/blob/main/dm_control/locomotion/walkers/assets/dog_v2/create_skin.py
i passed to the function create an .obj file the mesh of the skin i want to create and the hand.xml file.
It all works and creates the hand_skin.skn file as i named it, but when i import the complete xml file to mujoco including the skin it gives me this error : 

insufficient data in SKN file bone 7

I really don't understand how to solve this problem and hopeful that someone can help.
Thanks

**Comment by yuvaltassa** [comment]
The error is [here](https://github.com/google-deepmind/mujoco/blob/3d1d1d07183432b29b5b4601edb7fdb497feb41e/src/user/user_mesh.cc#L2242) or [here](https://github.com/google-deepmind/mujoco/blob/3d1d1d07183432b29b5b4601edb7fdb497feb41e/src/user/user_mesh.cc#L2281). I recommend building from source and stepping with a debugger.

**Comment by PiNo010** [comment]
the error is [here](https://github.com/google-deepmind/mujoco/blob/3d1d1d07183432b29b5b4601edb7fdb497feb41e/src/user/user_mesh.cc#L2281). so i don't understand if the problem is how i exported the mesh from blender (.obj file) or what i do with the model before calling the creat_skin function.
Maybe someone can explain to me how to export correctly all the files
